FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDeviceAgnosticTests.m

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2013, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+
+@interface FBSnapshotTestCaseDeviceAgnosticTests : FBSnapshotTestCase
+
+@end
+
+@implementation FBSnapshotTestCaseDeviceAgnosticTests
+
+- (void)setUp
+{
+    [super setUp];
+    self.deviceAgnostic = YES;
+    self.recordMode = NO;
+}
+
+- (void)testViewSnapshot
+{
+    UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
+    redView.backgroundColor = [UIColor redColor];
+    FBSnapshotVerifyView(redView, nil);
```

Comment:
Sorry, I had to look at it a bit to figure out what is going on and there are a couple of issues:

1. The build script that I added on travis is building and testing both [iPhone 5 and iPhone 6](https://github.com/facebook/ios-snapshot-test-case/blob/master/build.sh#L25-L26), so we have to record the image for both of these devices.  

2. The [build script](https://github.com/facebook/ios-snapshot-test-case/blob/master/build.sh#L7-L11) runs the FBSnapshotControllerTests and in particular [testFailedImageWithDeviceAgnosticShouldHaveModelOnName](https://github.com/facebook/ios-snapshot-test-case/blob/master/FBSnapshotTestCaseTests/FBSnapshotControllerTests.m#L69). These tests don't have an application associated with, so we hit the assertion. Maybe we can make the window a parameter and simply write tests to make sure that FBDeviceAgnosticNormalizedFileName is working as expected.